service/src/test/java/bio/terra/workspace/service/workspace/WorkspaceServiceTest.java

```diff
@@ -548,6 +549,36 @@ void createMcWorkspaceUndoSteps() {
         () -> workspaceService.getWorkspace(request.getWorkspaceId()));
   }
 
+  @Test
+  void testDeleteWorkspaceProperties() {
+    // Create one workspace with properties
+    Map<String, String> propertyMap = new HashMap<>();
+    propertyMap.put("foo", "bar");
+    propertyMap.put("xyzzy", "plohg");
+
+    Workspace request = defaultRequestBuilder(UUID.randomUUID()).properties(propertyMap).build();
+    workspaceService.createWorkspace(request, USER_REQUEST);
+    UUID workspaceUuid = request.getWorkspaceId();
+    var lastUpdatedDate = workspaceActivityLogDao.getLastUpdatedDate(workspaceUuid);
+    assertTrue(lastUpdatedDate.isPresent());
+
+    List<String> propertyKeys = new ArrayList<>();
```

Comment:
nit: this can be a oneline: new ArrayList<>(
            Arrays.asList("foo", "foo1"))

Reply:
great!